foundry/foundry.py

```diff
@@ -854,9 +852,52 @@ def toTorch(self, raw=None, split=None):
                 for key in df.keys():
                     arr.append(df[key].values)
 
-            return FoundryDataset_Torch(inputs, targets)
+            return FoundryDatasetAsTorchDataset(inputs, targets)
         else:
             raise NotImplementedError
+
+    def to_tensorflow(self, split=None):
+        """Convert Foundry Dataset to a Tensorflow Sequence
+
+        Arguments:
+            split (string): Split to create Tensorflow Sequence on.
+                    **Default:** ``None``
+
+        Returns: (FoundryDatasetAsTensorflowSequence) Tensorflow Sequence of all the data from the specified split
+
+        """
+        raw = self.load_data(as_hdf5=False)
```

Comment:
Move the import statement for the TF data loader into this function so that Foundry won't load Tensorflow until needed. Imports not at the top is a non-standard practice, but useful when adding functions which rely on optional requirements (e.g., Pandas does this: https://github.com/pandas-dev/pandas/blob/v1.4.3/pandas/core/generic.py#L2190-L2353)

Reply:
I can't because the Data Loader extends a TF Sequence.